src/dm_logger.vala

```diff
@@ -419,6 +432,7 @@ namespace DMLogger
       }
       add_to_log_buffer( &this.trace_level, sizeof( uint16 ) );
       add_to_log_buffer( &this.message_id, sizeof( int64 ) );
+      add_to_log_buffer( &this.component, sizeof( string ) );
```

Comment:
Das geht so nicht.
Da schreibst du ja nur den Pointer des Strings in das File.
Bitte verwende `write_log_string` (so wie nur 4 Zeilen oder so darunter)